### DISCUSSION TITLE
Can Riva pipeline work with custom fine-tuned P&C model?

### DISCUSSION THREAD

**Discussion by ti3x** [NVIDIA-NeMo/NeMo#5601] [Q&A] [answered]
Was reading on https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html#beginning-end-of-utterance-detection
There seemed to be guides on how to fine-tune a custom P&C model for ASR on both TAO and NeMo, but a bit unclear on how to integrate it with Riva pipeline.

**Comment by virajkarandikar** [comment, marked as answer]
See https://docs.nvidia.com/deeplearning/riva/user-guide/docs/nlp/nlp-customizing.html
